packages/js/src/utils/inline.ts

```diff
@@ -73,7 +73,13 @@ export function postProcessInlinedDependencies(
         inlineDependency.buildOutputPath ||
         join(outputPath, inlineDependency.root);
       const destDepOutputPath = join(outputPath, inlineDependency.name);
-      movePackage(depOutputPath, destDepOutputPath);
+      const isBuildable = !!inlineDependency.buildOutputPath;
+
+      movePackage(
```

Comment:
Maybe better to inline the copy/move logic to avoid additional indirection.

like

```ts
const isBuildable = !!inlineDependency.buildOutputPath;

if (isBuildable) {
  copySync(...);
} else {
  moveSync(...);
}
```